### PR TITLE
save work-in-progress indexes every 60s

### PR DESCRIPTION
**Pull request by staltz**
This PR is now pending on https://github.com/ssb-ngi-pointer/async-append-only-log/pull/24 getting merged first and released.

### CODE REVIEW

File: test/slow-save.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+const rimraf = require('rimraf')
+const mkdirp = require('mkdirp')
+
+const dir = '/tmp/jitdb-slow-save'
+rimraf.sync(dir)
+mkdirp.sync(dir)
+
+var keys = ssbKeys.loadOrCreateSync(path.join(dir, 'secret'))
+
+prepareAndRunTest('wip-index-save', dir, (t, db, raf) => {
```
Comment (arj03) [inline test/slow-save.js:16]: I guess we can remove the wip from the names here?

**Review by arj03 — APPROVED** [review]

File: test/slow-save.js (hunk 1 above)
Comment (arj03) [reply test/slow-save.js:16]: Just a minor thing, the PR overall looks good.

Comment (arj03) [reply test/slow-save.js:16]: Oh, I'm slow today. Scratch this.

Comment (staltz) [reply test/slow-save.js:16]: Yeah it looks a bit weird as a test name, I also got a tiny bit confused sometimes

**Pull request merged**